### PR TITLE
require a cloud directory

### PR DESCRIPTION
**Pull request by noamcattan**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -110,11 +111,17 @@ func (i *InitActionCmd) Run(ctx context.Context, client *githubClient, current r
 	if err != nil {
 		return err
 	}
-	// If dir name is not set by the user, and there are dirs in the cloud, prompt the user to choose one.
-	if i.DirName == "" && len(dirNames) > 0 {
-		if err = i.setDirName(dirNames); err != nil {
+	if len(dirNames) == 0 {
+		return errors.New("no migration directories found in your organization")
```
Comment (noamcattan) [inline main.go:115]: We can add a link to the docs here later

File: main.go [hunk 2]
```diff
@@ -110,11 +111,17 @@ func (i *InitActionCmd) Run(ctx context.Context, client *githubClient, current r
...
+	}
+	switch {
```
Comment (a8m) [inline main.go:117]: You can add it as another case.

**Review by a8m — DISMISSED** [review]

File: .github/workflows/ci-go.yaml [hunk 3]
```diff
@@ -35,35 +35,45 @@ jobs:
...
+      - name: Install extension
         run: |
           gh extension install .
-      - name: run GitHub atlas extension
+      - name: Run GitHub atlas extension
         run: |
-          gh atlas init-action -R ${{ secrets.TEST_REPO }} --token=${{ secrets.TEST_ATLAS_TOKEN }} --dir-name="test-sqlite" --driver="sqlite" "sqlite_dir"
-      - name: validate pull request is created
+          gh atlas init-action -R ${{ secrets.TEST_REPO }} --token=${{ secrets.TEST_ATLAS_TOKEN }} --dir-name="sync-action-integration-test" --driver="sqlite" "sqlite_dir"
+      - name: Run Gituh atlas extension with invalid dir
```
Comment (dorav) [inline .github/workflows/ci-go.yaml:59]:
```suggestion
      - name: Run GitHub atlas extension with invalid dir
```

File: main.go (hunk 2 above)
Comment (dorav) [reply main.go:117]: add what?

**Review by dorav — APPROVED** [review]

**Review by dorav — APPROVED** [review]

**Pull request merged**